setup.cfg

```diff
@@ -10,7 +10,7 @@ exclude = docs,
 test = pytest
 
 [tool:pytest]
-addopts = --verbose -x
+addopts = --verbose
```

Comment:
(if we later revert the pandas2 thing, it would be strange to get this -x reintroduced, but no biggie..)